app/views/facility_orders/_order_table.html.haml

```diff
@@ -29,9 +29,19 @@
         %td.order_date
           = od.ordered_at&.strftime("%m/%d/%Y")
 
-      %td.currency.timeinput= od.reservation.try(:duration_mins)
+      - daily_booking_product = od.product.is_a?(Instrument) && od.product.daily_booking?
```

Comment:
This check is repeated a couple of times, and I feel it might be used again in the future. Should we add it as a method in the product model?